### PR TITLE
i18n(es): Translate `dynamically-importing-images`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Translated `dynamically-importing-images` to spanish

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/recipes/dynamically-importing-images.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: Dinámicamente Importa Imágenes
```
Comment (Waxer59) [inline src/content/docs/es/recipes/dynamically-importing-images.mdx:2]:
```suggestion
title: Importa Imágenes Dinámicamente 
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**